css-typed-om/Overview.bs

```diff
@@ -990,6 +1008,24 @@ The following are the arithmetic operations you can perform on dimensions:
         is set to |result|.
 </div>
 
+<div algorithm="CSSNumericValue.type()">
+    The <dfn method for=CSSNumericValue>type()</dfn> method
+    returns a representation of the [=type=] of |this|.
+
+    When called, it must perform the following steps:
+
+    1. Let |result| be a new {{CSSNumericType}} with all attributes set to `null`.
```

Comment:
Can I omit the "with all attributes set to null" since it's probably the default behaviour?

Reply:
Yeah.